src/preferences/dialog/dlgprefrecord.cpp

```diff
@@ -117,21 +117,30 @@ DlgPrefRecord::DlgPrefRecord(QWidget* parent, UserSettingsPointer pConfig)
 
     slotApply();
     // Make sure a corrupt config file won't cause us to record constantly.
+    // TODO: What does this do exactly? I thougth it stopped the recording when showing
+    // the preferences page but it does not (and i would preffer that keeps not stopping it).
     m_pRecordControl->set(RECORD_OFF);
```

Comment:
I am not sure if I understand you correctly. Do you mean that I have to remove the comments I added, or that I have to remove the "set", because it is useless?
In fact, I am not sure if the initial value is set anywhere. Probably recordingmanager should do it in its constructor.